network/acp118/handler.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package acp118
```

Comment:
Should this package live in `network/p2p` rather than just `network`? It seems fairly similar to `gossip`